conf/application.conf

```diff
@@ -93,25 +93,33 @@ microservice {
       host = localhost
       port = 8500
     }
-    eis {
+    hawk {
         protocol = http
         host = localhost
-        port = 10903
+        port = 10908
```

Comment:
Leave this as 10903 so we use the static stubs by default

Reply:
Done ✅